src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -352,12 +363,21 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
       },
       trajectory: {
         inputType: 'selection',
-        selectionTypes: ['segment', 'path'],
+        selectionTypes: ['segment'],
```

Comment:
Right now we can only select segments, not paths, so it doesn't make sense to have it set to both.